Monika After Story/game/script-topics.rpy

```diff
@@ -2887,6 +2893,15 @@ label monika_love:
     $ persistent._mas_monika_lovecountertime = datetime.datetime.now()
     return
 
+label monika_love_reaction_event:
+    #Another Monday morning, eh, [player]?  ->  I love you so much, [player]~
+    if persistent._mas_monika_love_reaction_event == 1:
```

Comment:
Didn't you just evaluate this such that it has to not be zero (only other possibility is 1) to get here? This check is unnecessary.

Reply:
@multimokia it's not a true/false check, it's being treated as integer IDs. Set the var to a number, check the appropraite number in the love reaction event and show whatever dialogue.

It would be better if this uses a dict to organize IDs with associated labels/events.

Reply:
Yeah that would then allow expansion to other greets and such.

Reply:
Feel free to enhance it, this is a working prototype.

I am new to python and thus not as experienced as you guys.